.tool-versions

```diff
@@ -0,0 +1 @@
+golang 1.19
```

Comment:
any reason not to update to latest (1.20)?

Reply:
just matching the version in go.mod which itself is pinned to the [version used by cosmos-sdk](https://github.com/cosmos/cosmos-sdk/blob/v0.46.11/go.mod#L1). historically, the go version that kava is built with was important so i figured i'd keep parity with the repo itself